src/ui/components/Passport/Passport.module.css

```diff
@@ -1,8 +1,8 @@
 .PassportBoxWrapper {
   position: relative;
-  background-color: powderblue;
+  background-color: #dde7ef;
```

Comment:
Hard-coded colors seem like they'd break our light/dark theme?

Reply:
Currently the passport is always the same blue color with passport pattern.